extensions/azure/data-plane/data-factory/src/test/java/org/eclipse/dataspaceconnector/azure/dataplane/azuredatafactory/AzureDataFactoryCopyIntegrationTest.java

```diff
@@ -145,6 +162,22 @@ void transfer_success(
                 .isEqualTo(randomBytes.length);
     }
 
+    private void setSecret(Account consumerStorage, Vault vault, String secretName) {
+        var expiryTime = OffsetDateTime.now().plusMinutes(5);
```

Comment:
SLA for ADF is to start an activity within 4 minutes. Consider extending?

Reply:
Thanks, good catch. Updated to 8 minutes.